From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: violetc <58360096+s-yh-china@users.noreply.github.com>
Date: Wed, 17 Aug 2022 16:54:54 +0800
Subject: [PATCH] Async Pathfinding

This patch is Powered by Pufferfish(https://github.com/pufferfish-gg/Pufferfish)
But Pufferfish patch was ported downstream from the Petal fork

diff --git a/src/main/java/top/leavesmc/leaves/LeavesConfig.java b/src/main/java/top/leavesmc/leaves/LeavesConfig.java
index 4546a98ae419ff5f647e79455b7d36b21bdb92a8..5a75b8331c2ca804fa61002d9a2e0a8fcf786d3c 100644
--- a/src/main/java/top/leavesmc/leaves/LeavesConfig.java
+++ b/src/main/java/top/leavesmc/leaves/LeavesConfig.java
@@ -326,7 +326,7 @@ public final class LeavesConfig {
     // only config now
     public static boolean asyncPathfinding = false;
     private static boolean asyncPathfindingLock = false;
-    private static void asyncPathfinding() {
+    private static void asyncPathfinding() { // void
         if (!asyncPathfindingLock) {
             asyncPathfinding = getBoolean("settings.performance.async-pathfinding", asyncPathfinding);
             asyncPathfindingLock = true;
